src/VirtoCommerce.SimpleExportImportModule.Web/Scripts/resources/resources.js

```diff
@@ -0,0 +1,656 @@
+angular.module('virtoCommerce.simpleExportImportModule')
+    .factory('virtoCommerce.simpleExportImportModule.import', ['$resource', '$q', function ($resource, $q) {
+
+        var mockData = {
+            totalCount: 10000,
+            records: [
+                {
+                    "productId": "6d4f5835f5f8459590da29528b175ff7",
```

Comment:
We already have a backend for preview. Isn't it? Why we need this mock data?

Reply:
The backend was in another branch.

Reply:
And? We still need this mock data here?

Reply:
And it is in progress